modules/src/wiki.py

```diff
@@ -40,7 +39,7 @@ def process(input, entities):
                     }
                 }
                 buttons.add_postback('Wikipedia Summary', payload)
-                template.add_element(title=data.title, item_url=data.url, image_url=image_url,
+                template.add_element(title=data.title, item_url=data.url, image_url=data.images[0],
```

Comment:
The `images` list might be empty. In that case, let's default to the current image_url.